2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -41,7 +41,7 @@ function createBookList(books) {
               list-style:none`;
 
 
-  for (let i = 0; i < books.length; i++) {
+  for (const v of books) {
```

Comment:
Nice! All I'd suggest now is to avoid single-letter variables. I'd use `const book` instead of `const v` :)